instrumentation/opentelemetry-instrumentation-threading/README.rst

```diff
@@ -0,0 +1,23 @@
+OpenTelemetry threading Instrumentation
+=======================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-instrumentation-threading.svg
+   :target: https://pypi.org/project/opentelemetry-instrumentation-threading/
+
+This library provides instrumentation for the `threading` module to ensure that
+the OpenTelemetry context is propagated across threads.
```

Comment:
Can you add a note that this instrumentation does not produce any telemetry?

Reply:
Done